proposals/README.md

```diff
@@ -0,0 +1,6 @@
+# Azure Kinect Sensor SDK Proposals
+
+New features proposals are living documents describing the current thinking about a given feature. Proposals are the way for Azure Kinect team  to communicate to the public upcoming new feature work and get the feedback.
+At this time, all proposals will be created by Microsoft Azure Kinect team for the most desired features, but if you have a great idea for a proposal, please submit it on [User Voice](https://feedback.azure.com/forums/920053).
```

Comment:
When will MS kick off a new proposal?
How much time do we give everyone to review?